src/core.js

```diff
@@ -17,6 +17,15 @@ var Plotly = require('./plotly');
 // package version injected by `npm run preprocess`
 exports.version = '1.16.2';
 
+// inject promise polyfill
+require('es6-promise').polyfill();
```

Comment:
will be gone in `v2` https://github.com/plotly/plotly.js/issues/420#issue-148094498